Qiy Scheme Releases.md

```diff
@@ -3,22 +3,27 @@
 ## Abstract
 The Qiy Scheme defines a framework for individual users, companies and governmental organisations to safely control and exchange personal data. This document describes what the requirements are for new releases of the Qiy Scheme and how these new releases are managed.
 
-# Content
-
 # 1 Qiy Scheme: new releases
-This document is a policy document which describes what the requirements are for new releases of the Qiy Scheme and how these new releases are managed.
+This document is a policy document which describes the requirements for new releases of the Qiy Scheme and how these new releases are managed.
+
+# 2 Versioning
+The Supervisory Authority of the Qiy Foundation decides upon the release of new versions of the Qiy Scheme:
```

Comment:
The Executive Board of the Qiy Foundation decides upon the release of new versions of the Qiy Scheme: